planetiler-core/src/test/java/com/onthegomap/planetiler/PlanetilerTests.java

```diff
@@ -687,7 +687,7 @@ void testSkipFill() throws Exception {
         .setBufferPixels(4)
     );
 
-    assertEquals(481, results.tiles.size());
+    assertEquals(130, results.tiles.size());
```

Comment:
@bdon this was fixed by #279 - you can revert this and rebase and it should be fine now.